src/app.ts

```diff
@@ -233,6 +236,39 @@ try {
     process.exit(1);
 }
 
+container.register(
+    'storageAdapter',
+    asFunction(() => {
+        const bucketName = process.env.GCP_BUCKET_NAME || '';
+        return new GCPStorageAdapter(
+            bucketName,
+            process.env.GCP_STORAGE_EMULATOR_HOST ?? undefined,
+        );
```

Comment:
As we add more storage adapters, I imagine this block could select the right adapter based on config